src/main/java/com/stevedutch/assignment14/domain/User.java

```diff
@@ -19,42 +19,37 @@
 @Table(name = "users") // changed into users
 public class User {
 
-//	@SerializedName("username") in Jackson it would be @JsonProperty("username")
 	@Column(unique = true)
 	public String username;
 
 //	@SerializedName("userId")
 	@Id
 	@GeneratedValue(strategy = GenerationType.IDENTITY)
-	@Column(name = "user_id")
-	private Long userId;
+	private Long id;
 
 	@JsonManagedReference
-//	@SerializedName("messages")
 	@OneToMany(mappedBy = "user")
 	private List<Message> messages = new ArrayList<>();
 
 	public User(String username, List<Message> messages) {
-		super();
```

Comment:
for the constructors, there's no need to call super(). The superclass for User is Object (since you're not defining any parent classes) and all Java classes inherit all methods and fields from Object by default, so no need to call super()